src/python/pants/backend/native/subsystems/binaries/llvm.py

```diff
@@ -90,7 +90,7 @@ def linker(self, platform):
       path_entries=self.path_entries,
       exe_filename=platform.resolve_platform_specific(
         self._PLATFORM_SPECIFIC_LINKER_NAME),
-      library_dirs=[],
+      runtime_library_dirs=[],
       linking_library_dirs=[],
```

Comment:
Im slightly confused on the difference here between `runtime_library_dirs` and `linking_library_dirs`. Is the latter for static libs?

Reply:
`Directories containing shared libraries that must be on the runtime library search path.` (this is `LD_LIBRARY_PATH` or `DYLD_LIBRARY_PATH`) vs `Directories to search for libraries needed at link time.` (this is `LIBRARY_PATH`, it's for both static and shared libs)